src/index.html

```diff
@@ -9,6 +9,16 @@
   <link rel="stylesheet" href="./styles/main.scss">
 </head>
 <body>
-  <h1>Stop watch</h1>
+  <div class="container">
+    <div class="container__clock">
+      <div class="arrows">
+        <div class="arrows__arrow arrows__arrow--minute"></div>
+        <div class="arrows__arrow arrows__arrow--second"></div>
+        <div class="arrows__arrow arrows__arrow--center"></div>
```

Comment:
check the previous commet 
<img width="750" alt="image" src="https://user-images.githubusercontent.com/102411130/212266199-a54377c8-5f59-4c9e-9294-995940d96d6d.png">